corehq/apps/style/decorators.py

```diff
@@ -52,3 +52,15 @@ def wrapped(request, *args, **kwargs):
             return fn(request, *args, **kwargs)
         return wrapped
     return decorate
+
+
+def upgrade_knockout_js():
```

Comment:
i actually think this is kind of an antipattern: https://github.com/dimagi/commcare-hq/pull/8499/files for an example.

i believe that example allows us not to use `@method_decorator`